tests/php/test_class.jetpack-gutenberg.php

```diff
@@ -129,4 +130,18 @@ function test_plugin_is_not_available_when_not_registered_returns_missing_module
 		$this->assertFalse( $availability['tomato']['available'], 'Availability is not false exists' );
 		$this->assertEquals( $availability['tomato']['unavailable_reason'], 'missing_module', 'unavailable_reason is not "missing_module"'  );
 	}
+
+	function test_get_available_extensions() {
+		$extensions = Jetpack_Gutenberg::get_available_extensions( $this->get_extensions_whitelist() );
+		$this->assertInternalType( 'array', $extensions );
+		$this->assertNotEmpty( $extensions );
+		$this->assertContains( 'onion', $extensions );
+
+		update_option( 'jetpack_extensions_exclusions', array( 'onion' ) );
+
+		$extensions = Jetpack_Gutenberg::get_available_extensions( $this->get_extensions_whitelist() );
+		$this->assertInternalType( 'array', $extensions );
+		$this->assertNotEmpty( $extensions );
+		$this->assertNotContains( 'onion', $extensions );
```

Comment:
Should we also add a test for setting the option back to an empty array, and verifying that this is recognized properly in the same way as when no option exists?

Reply:
That's a good question. I'm not sure that it's necessary though since we default the `get_option()` to return `array()` when the option does not exist.